### ISSUE TITLE
unable to deploy to georli  followed the readme steps

### ISSUE THREAD

**Issue by balook** [mountainprotocol/tokens#40]
Im getting error 

```
_ Help us improve Hardhat with anonymous crash reports & basic usage data? (Y/n) · y
Error: Missing OpenZeppelin Platform API key and secret in hardhat config. Add the following to your hardhat.config.js configuration:
platform: {
  "apiKey": "YOUR_API_KEY",
  "apiSecret": "YOUR_API_SECRET"
}

```

i used command

npx hardhat run scripts/deploy.ts --network goerli

**Comment by mattiascaricato** [comment]
hey @balook thanks for the feedback. We are adding the ability to deploy not only using Defender from OpenZeppelin but also using just hardhat from the terminal. You can check it out in following PR: #42

**Comment by mattiascaricato** [comment]
addressed in PR #42
